tutorial/06_provide_presimulated.ipynb

```diff
@@ -0,0 +1,363 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Providing presimulated data\n",
+    "\n",
+    "In this tutorial, we will demonstrate how one can obtain the posterior with `sbi` from data that been simulated beforehand (for example data that had been simulated on a compute cluster)."
```

Comment:
"In this tutorial, we will demonstrate how one can obtain the posterior with `sbi` using data that was generated beforehand (for example on a compute cluster)"